code/modules/projectiles/gun.dm

```diff
@@ -46,7 +46,7 @@
 /obj/item/weapon/gun/examine(mob/user)
 	..()
 	if(pin)
-		user << "It has a [pin] installed."
+		user << "It has [pin] installed."
```

Comment:
wouldn't this say "It has the electronic firing pin installed" , that looks kind of weird
why not put \an and capitalize the firing pin names?